addons/sys_core/fnc_processRadioSpeaker.sqf

```diff
@@ -25,14 +25,8 @@ private ["_okRadios", "_functionName"];
 params ["_unit","_playerRadios"];
 TRACE_2("",_unit,_playerRadios);
 
-private _radioId = _unit getVariable [QGVAR(currentSpeakingRadio), ""];
-if (_radioId == "") exitWith { false };
-
-// Debug #638
-if (isNil "_radioId") exitWith {
-    WARNING_1("reserved variable debug 1 - report on GitHub! [%1]",_radioId);
-    false
-};
+private _radioId = _unit getVariable QGVAR(currentSpeakingRadio);
+if (isNil "_radioId" || {_radioId == ""}) exitWith { false }; // #638 _radioId is nil
```

Comment:
Although this fixes it, i would be interested to know why it is "". Maybe add a comment that we should investigate further?

Reply:
Better question is why it is nil. Empty string is just the default value of getVariable.

Reply:
True. I would expand the comments nevertheless:)

Reply:
Yes, for sure. :)

Reply:
Expanded it with a probable source (only place where it gets set). However the issue lies deeper than that link of course.